aries-site/src/layouts/navigation/NavPage.js

```diff
@@ -16,21 +16,26 @@ const NavItem = ({ item, topic }) => {
   return (
     // Need to pass href because of: https://github.com/zeit/next.js/#forcing-the-link-to-expose-href-to-its-child
     <Link href={`/${topic}/${formattedItem}`} passHref>
-      <Box
-        direction="row"
-        margin={{ vertical: 'large' }}
-        gap={size !== 'small' ? 'large' : 'medium'}
-      >
-        {/* Adds placeholder icon for the meantime
-         * while we wait to get all the icons */}
-        {itemData.icon ? itemData.icon('xlarge') : iconsMap.branding('xlarge')}
-        <Box>
-          <Text weight="bold" size={size !== 'small' ? 'large' : 'medium'}>
-            {itemData.name}
-          </Text>
-          <Paragraph size="small">{itemData.description}</Paragraph>
+      {/* Needs to be <a> in DOM for web crawling: https://support.google.com/webmasters/answer/9112205?hl=en */}
+      <Button fill>
+        <Box
+          direction="row"
+          margin={{ vertical: 'large' }}
+          gap={size !== 'small' ? 'large' : 'medium'}
+        >
+          {/* Adds placeholder icon for the meantime
+           * while we wait to get all the icons */}
+          {itemData.icon
+            ? itemData.icon('xlarge')
+            : iconsMap.branding('xlarge')}
```

Comment:
Is this condition still needed? Opportunity to eliminate while we are in here?

Reply:
The only reason I would leave it is if we think there'd be a case where a page didn't have an icon yet and it we want to leave it as a fallback -- at least we'd have confidence that the page layout would look correct as pages get added.